tests/release/release.go

```diff
@@ -128,21 +128,21 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1108]test-release-service-happy-p
 				}
 
 				return prList.Items[0].HasStarted() && prList.Items[0].IsDone() && prList.Items[0].Status.GetCondition(apis.ConditionSucceeded).IsTrue()
-			}, avgPipelineCompletionTime, defaultInterval).Should(BeTrue())
+			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(BeTrue())
 		})
 
-		It("The Release should have succeeded.", func() {
+		It("makes sure that the Release should have succeeded.", func() {
 			Eventually(func() bool {
 				release, err := framework.ReleaseController.GetRelease(releaseName, devNamespace)
 				if err != nil || release == nil {
 					return false
 				}
 
 				return release.IsDone() && meta.IsStatusConditionTrue(release.Status.Conditions, "Succeeded")
-			}, avgPipelineCompletionTime, defaultInterval).Should(BeTrue())
+			}, releaseCreationTimeout, defaultInterval).Should(BeTrue())
 		})
 
-		It("The Release should reference the release PipelineRun.", func() {
+		It("the Release should reference the release PipelineRun.", func(ctx SpecContext) {
```

Comment:
this one still needs fixing the description.

Reply:
> this one still needs fixing the description.

Updated accordingly, thanks